assets/component-price.css

```diff
@@ -64,10 +64,15 @@
 }
 
 .price--sold-out .price__badge-sold-out,
-.price--on-sale .price__badge-sale {
+.price--on-sale .price__badge-sale,
+.volume-pricing--sale-badge .price__badge-sale {
   display: inline-block;
 }
 
+.volume-pricing--sale-badge .price__badge-sale {
+  margin-left: 0.5rem;
+}
+
 .price--on-sale .price__sale {
```

Comment:
Shouldnt we show the discount in the card too?

https://screenshot.click/19-54-2nj6o-45zfa.png

Reply:
@edmund-teh ?

Reply:
Should it look something like this?
![Uploading Screenshot 2023-07-19 at 11.03.03 AM.png…]()

Reply:
@eugenekasimov the styling looks great! But I think we should double-check with @nicklepine, if this is exactly what we intend to show.

Reply:
We discussed it with @nicklepine, all looks good.